opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -630,8 +637,10 @@ def _from_env_if_absent(
     max_links=SpanLimits.UNSET,
     max_event_attributes=SpanLimits.UNSET,
     max_link_attributes=SpanLimits.UNSET,
+    max_attribute_length=SpanLimits.UNSET,
```

Comment:
For some reason this limit is the only one in the specs to have notes. Negative values are invalid apparently? What do we do in those cases.

Reply:
I'm sure that note applies to all limits and it was just a mistake by the author. I'll take care of negative/unset values in a new PR (created issue: https://github.com/open-telemetry/opentelemetry-python/issues/2052)